spec/requests/posts_spec.rb

```diff
@@ -2,16 +2,24 @@
 
 RSpec.describe 'Posts', type: :request do
   describe 'GET /index' do
-    it 'returns http success' do
-      get '/posts/index'
-      expect(response).to have_http_status(:success)
+    before(:example) { get user_posts_path(1) }
+    it 'should be success' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it "should render 'index' template" do
+      expect(response).to render_template('index')
+    end
+
+    it 'should return the correct placeholder text' do
+      expect(response.body).to include('This is the place for all posts.')
     end
   end
 
   describe 'GET /show' do
-    it 'returns http success' do
-      get '/posts/show'
-      expect(response).to have_http_status(:success)
+    before(:each) { get user_post_path(1, 2) }
+    it 'returns success' do
+      expect(response).to have_http_status(:ok)
     end
   end
 end
```

Comment:
- [ ] You have one failing test, kindly update it
![image](https://user-images.githubusercontent.com/62705947/197761694-aa547165-2104-4efd-ac4c-55da1d0a78f6.png)